### PR TITLE
Paging support in sub resource lookup

### PR DESCRIPTION
**Pull request by alexeiser**
Update the get_sub_resource to automatically fetch ALL children instead of stopping at the PAGE limit.

### CODE REVIEW

File: tests/test_course.py [hunk 1]
```diff
@@ -33,5 +33,5 @@ def test_module_complete(self, api_mock):
             None,
             'modules',
             'fg2',
-            {'CourseId': course.Id, 'UpdatedAt': '/Date(1478785811390)/'}
+            ANY
```
Comment (alexeiser) [inline tests/test_course.py:36]: On my python env, the keys are inverted - moving to ANY since the resulting object is not critical.

**Comment by alexeiser** [comment]
> closing as no unit tests

Sorry - should have ran the unit tests - updated.

Please re-review.

**Comment by charliequinn** [comment]
hey @alexeiser thanks for the PR much appreciated. The tests are failing however I can fix these

**Review by charliequinn — APPROVED** [review]

**Comment by charliequinn** [comment]
released in version 1.2.0

**Comment by alexeiser** [comment]
> The tests are failing however I can fix these.

Thanks, I couldn’t test those because they relied on your Litmos api key. Thanks for addressing them!

**Pull request merged**